app/views/lead_providers/guidance/npq_usage.html.erb

```diff
@@ -1,7 +1,11 @@
 <h2 class="govuk-heading-l">Contents</h2>
 
 <ul class="govuk-list govuk-list--bullet">
-  <li><a href="#continuing-the-npq-application-process" class="govuk-link">Continuing the NPQ application process</a></li>
+  <li><a href="#continuing-the-npq-registration-process" class="govuk-link">Continuing the NPQ registration process</a></li>
```

Comment:
This breaks an rspec test, need to update copy in that

Reply:
Yes. Spotted and fixed that and duplicated it for the new section too.